README.md

```diff
@@ -89,7 +89,7 @@ Please see the dedicated [README.md](tools/docker/README.md) for information on
 | iceoryx_dds                             | 4    | 4                     |                                     |
 | iceoryx_meta                            | 5    | 5                     |                                     |
 | iceoryx_posh                            | 1+, 2| 2                     | Will be split into separate targets |
-| iceoryx_utils                           | 1+   | 2                     |                                     |
+| iceoryx_hoofs                           | 1+   | 2                     |                                     |
 | iceoryx_introspection                   | 5    | 5                     |                                     |
```

Comment:
The alphabetical ordering is now even more in a muddle. Shall we order them semantically from building blocks to extensions?